tests/driver_mrf24j40/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = driver_mrf24j40
```

Comment:
Yes. Please remove it. One can use `gnrc_networking` or `default`. Later #5286 will unify these tests anyway.